code/game/objects/items/devices/text_to_speech.dm

```diff
@@ -0,0 +1,28 @@
+/obj/item/weapon/text_to_speech
```

Comment:
Why is this pathed as a weapon, in the devices folder?

Reply:
Whoops. I wrote it as a weapon initially and planned to make it a device. Let me fix that real quick.